src/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux';
+import books from './book';
+
+const rootReducer = state => {
+  const booksReducer = books(state);
+  return combineReducers({
+    booksReducer,
+  });
+};
```

Comment:
If you pass the initial state as the second argument to `createStore`, Redux will take care of this and you can simplify your `rootReducer`:

```js
const rootReducer = combineReducers({
    booksReducer: book,
});
```